docs/lib/predictions/fragments/android/getting-started.md

```diff
@@ -0,0 +1,171 @@
+## Prerequisites
+
+* [Install and configure the Amplify CLI](~/cli/start/install.md)
+
+```bash
+npm install -g @aws-amplify/cli
+amplify configure
+```
+
+## Automated Configuration
+
+The Amplify CLI helps you to create the appropriate IAM permissions to hit the AWS machine learning APIs. As part of the automated setup, the CLI helps you provision resources that leverage AWS AI/ML services.
+
+To create a project with the Predictions category, run the following command:
+
+1. Run `amplify init` command as shown:
+
+```console
+$ amplify init
+? Enter a name for the project: AmplifyPredictions
+? Enter a name for the environment: dev
+? Choose your default editor: IntelliJ IDEA
+? Choose the type of app that you're building: android
+? Where is your Res directory: app/src/main/res
+? Do you want to use an AWS profile? Yes
+? Please choose the profile you want to use: default
+```
+
+2. Add auth `amplify add auth` and choose `Default configuration`, allow users to sign in with `Email` and do not configure `advanced settings`
+
+3. Add predictions `amplify add predictions`
+    * Choose `Convert`
+    * Choose `Translate text into a different language`
+    * Provide a friendly name for your resource: `transTextSample`
+    * What is the source language? `English`
+    * What is the target language? `Italian`
+    * Who should have access? `Auth and Guest users`
+
+4. Run `amplify push` to create the resources in the cloud.
+
+When your backend is successfully updated, verify that `amplifyconfiguration.json` and `awsconfiguration.json` are properly updated inside the `res/raw` directory. Your new configuration file `amplifyconfiguration.json` will contain your default project `region` value as well as any necessary configuration options for each predictions method (i.e. target language and source language for translate, etc).
+
+
+## Manual Configuration
+
+If you have already created the resources in the cloud and would like to take advantage of those existing resources but still use the Amplify library in Android Studio, please follow the directions below:
+
+Create the file `amplifyconfiguration.json`:
+```
+touch amplifyconfiguration.json
+```
+
+Copy the contents over and update the values for the specific predictions method you are looking to use:
```

Comment:
Hmm, instead of touch and then a blob, why don't you just tell them explicit to open the file in Android Studio, and copy and paste the whole thing?

Reply:
Ah! you are right. It was a miss on my part while copying the instruction from iOS, but it would certainly be more helpful to instruct them to use Android Studio to directly create the files where they are required.